docs/website/versioned_docs/version-3.0.0/user-guides/quickstart.md

```diff
@@ -389,6 +486,204 @@ Your Devfile has been successfully deployed
 
 Your application has now been deployed to the Kubernetes cluster with Deployment, Service, and Ingress resources.
 
+Test your application by visiting the `DOMAIN_NAME` variable that you had set in the `devfile.yaml`.
+
+  </TabItem>
+  <TabItem value="dotnet" label=".NET">
+
+#### 1. Containerize the application
+
+In order to deploy our application, we must containerize it in order to build and push to a registry. Create the following `Dockerfile` in the same directory:
+
+```dockerfile
+FROM registry.access.redhat.com/ubi8/dotnet-60:6.0 as builder
+WORKDIR /opt/app-root/src
+COPY --chown=1001 . .
+RUN dotnet publish -c Release
+
+
+FROM registry.access.redhat.com/ubi8/dotnet-60:6.0
+EXPOSE 8080
+COPY --from=builder /opt/app-root/src/bin /opt/app-root/src/bin
+WORKDIR /opt/app-root/src/bin/Release/net6.0/publish
+CMD ["dotnet", "app.dll"]
+```
+
+#### 2. Modify the Devfile
+
+Let's modify the `devfile.yaml` and add the respective deployment code.
+
+`odo deploy` uses Devfile schema **2.2.0**. Change the schema to reflect the change:
+
+```yaml
+# Deploy "kind" ID's use schema 2.2.0+
+schemaVersion: 2.2.0
+```
+
+Add the `variables` section:
+
+```yaml
+# Add the following variables code anywhere in devfile.yaml
+# This MUST be a container registry you are able to access
+variables:
+  CONTAINER_IMAGE: quay.io/MYUSERNAME/dotnet-odo-example
+  RESOURCE_NAME: my-dotnet-app
+  CONTAINER_PORT: "8080"
+  DOMAIN_NAME: dotnet.example.com
+```
+
+Add the commands used to deploy:
+
+```yaml
+# This is the main "composite" command that will run all below commands
+- id: deploy
+  composite:
+    commands:
+    - build-image
+    - k8s-deployment
+    - k8s-service
+    - k8s-ingress
+    group:
+      isDefault: true
+      kind: deploy
+
+# Below are the commands and their respective components that they are "linked" to deploy
+- id: build-image
+  apply:
+    component: outerloop-build
+- id: k8s-deployment
+  apply:
+    component: outerloop-deployment
+- id: k8s-service
+  apply:
+    component: outerloop-service
+- id: k8s-ingress
+  apply:
+    component: outerloop-ingress
+```
+
+Add the Kubernetes Service and Ingress inline code to `components`:
+```yaml
+components:
+
+# This will build the container image before deployment
+- name: outerloop-build
+  image:
+    dockerfile:
+      buildContext: ${PROJECT_SOURCE}
+      rootRequired: false
+      uri: ./Dockerfile
+    imageName: "{{CONTAINER_IMAGE}}"
+
+# This will create a Deployment in order to run your container image across
+# the cluster.
+- name: outerloop-deployment
+  kubernetes:
+    inlined: |
+      kind: Deployment
+      apiVersion: apps/v1
+      metadata:
+        name: {{RESOURCE_NAME}}
+      spec:
+        replicas: 1
+        selector:
+          matchLabels:
+            app: {{RESOURCE_NAME}}
+        template:
+          metadata:
+            labels:
+              app: {{RESOURCE_NAME}}
+          spec:
+            containers:
+              - name: nodejs
```

Comment:
Since this is targeting .NET, maybe the container name could be more generic, independent of the programming language.